pkg/pool-weighted/test/gas/.hardhat-snapshots/[WeightedPool - Standard] add liquidity proportional

```diff
@@ -1 +1 @@
-179.3k
\ No newline at end of file
+179.8k
```

Comment:
:confused: I wasn't expecting this to happen; quite the opposite actually.
We shouldn't be loading the router extension here, and the router has less external methods now. Why is this going up?